multimedia/gpac/Portfile

```
@@ -20,9 +20,9 @@ long_description    GPAC is an Open Source multimedia framework for \
 
 homepage            http://gpac.wp.mines-telecom.fr/
 
-checksums           rmd160  9a95cf4b2bd0a98e995b21e13ba3ad1ccdb29246 \
-                    sha256  c5ac9846b313a718b030de629d9ed9c0a5db9fec40fdb4f803a571525eb63a8d \
-                    size    11406479
+checksums           rmd160  64c3fa909d3dbfa8a946f6365dc691a544c6db9f \
+                    sha256  7fb88cf996157fe2f70248f4667e4a2a0b5672a804b6d76fcbbd968f8c4dee67 \
+                    size    10354962
```

Please set `github.tarball_from archive`. This will change the checksums.
```suggestion
github.tarball_from archive
checksums           rmd160  1e19dba1a2770268f64947f6ba34fb458028764f \
                    sha256  3b0ffba73c68ea8847027c23f45cd81d705110ec47cf3c36f60e669de867e0af \
                    size    10352926
```